src/views/console/index.php

```diff
@@ -1,10 +1,8 @@
 <?php
 
-/**
- * @var Yiisoft\Requirements\RequirementsChecker $this
- * @var array $summary
- * @var array[] $requirements
- */
+/* @var $this Yiisoft\Requirements\RequirementsChecker */
```

Comment:
It's incorrect. See: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/var.html

Previous is OK.